deps/github.com/openshift/kubernetes/openshift-hack/rebase.sh

```
@@ -125,51 +124,64 @@ fi
-    XY=$(echo "$k8s_tag" | sed -E "s/v(1\.[0-9]+)\.[0-9]+/\1/")
-    ver=$(echo "$k8s_tag" | sed "s/\.//g")
-    link="https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-$XY.md#$ver"
-
-    # opens a web browser, because we can't properly create PRs against remote repositories with the GH CLI (yet):
-    # https://github.com/cli/cli/issues/2691
-    gh pr create \
-      --title "Bug $bugzilla_id: Rebase $k8s_tag" \
-      --body "CHANGELOG $link" \
-      --web
+if command -v gh &>/dev/null; then
+  XY=$(echo "$k8s_tag" | sed -E "s/v(1\.[0-9]+)\.[0-9]+/\1/")
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)